src/api/python/_api.py

```
@@ -925,22 +936,22 @@ def get_xfer_descs(
     ) -> nixlBind.nixlXferDList:
         # can add check for DLPack input
 
+        descs_len = None
+        try:
+            descs_len = len(descs)
+        except TypeError:
+            pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(TypeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(TypeError): ...`

(SIM105)